### PR TITLE
add network-module in function-corpping

### PR DESCRIPTION
**Pull request by novain1102**
Re: #https://github.com/cocos/cocos-engine/issues/12695

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/i18n/zh/localization.js [hunk 1]
```diff
@@ -904,9 +908,13 @@ module.exports = {
...
             description: "启用自定义渲染管线。",
         },
+        websocket: {
+            label: "WebSocket",
+            description: "对原生启用 WebSocket。其中iOS/macOS使用SocketRocket实现, Android使用OkHttp实现, Windows使用libwebsockets实现。",
+        },
         websocket_server: {
             label: "WebSocket Server",
-            description: "对原生启用 WebSocket Server。",
+            description: "对原生启用 WebSocket Server。注意: 必须同时启用 WebSocket。",
```
Comment (minggo) [inline editor/i18n/zh/localization.js:917]: Can automatically enable WebSocket if WebSocket Server is enabled?

Comment (PatriceJiang) [inline editor/i18n/zh/localization.js:917]: This can be handled in native/CmakeLists.txt

Comment (novain1102) [reply editor/i18n/zh/localization.js:917]: At present, there is no such mechanism on the editor. This dependency will be provided by cc.config.json. @yanOO1497 said that this part needs engine determined the fields and then editor could implement this mechanism.

Comment (novain1102) [reply editor/i18n/zh/localization.js:917]: track in issue: https://github.com/cocos/3d-tasks/issues/14052

**Review by minggo — APPROVED** [review]

**Review by yanOO1497 — APPROVED** [review]

**Review by PatriceJiang — APPROVED** [review]

**Pull request merged**